charts/redpanda/values.yaml

```diff
@@ -611,7 +613,7 @@ listeners:
         # enabled: true
         # The port used for external client connections
         port: 9094
-        # If undefined, `listeners.kafka.external.port` is used
+        # If undefined, `listeners.kafka.external.default.port` is used
```

Comment:
sanity check... If advertisedPorts is undefined, we advertise "9094" instead of the nodeport port?

Reply:
> Related to this, while creating this PR I realized that the schema registry listeners have a port defined, and this is unnecessary.

These additional listeners can have different TLS configurations so they do need their own port, iirc.